awsx/ec2/vpc.ts

```diff
@@ -604,3 +687,9 @@ export function validateNoGaps(vpcCidr: string, subnetSpecs: SubnetSpec[]) {
     `There are gaps in the subnet ranges. Please fix the following gaps: ${gaps.join(", ")}`,
   );
 }
+
+function rewrap(
+  x: pulumi.Output<pulumi.UnwrappedArray<schema.ResolvedSubnetSpecOutputs>>
+): pulumi.Output<schema.ResolvedSubnetSpecOutputs[]> {
+  return x as any;
```

Comment:
This is a code-smell and potentially unsafe, need to double-check how we get to need this.